### PR TITLE
Change examples to adhere to style guide

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Change examples to adhere to style guide

### Checklist
- [ x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: FORMATTING_GUIDE.md [hunk 1]
```diff
@@ -300,10 +300,10 @@ If you need to position an image or a code snippet within a list, use tabs to si
 **Example with images**
 
 ```
-1. To begin, select the rule in the Rule name column. The rule details pane opens.
+1. To begin, select the rule in the Rule name column. The rule details pane opens, as shown in the following image.
```
Comment (natebower) [inline FORMATTING_GUIDE.md:303]: Should "Rule name" be in bold as a UI element?

File: FORMATTING_GUIDE.md [hunk 2]
```diff
@@ -300,10 +300,10 @@ If you need to position an image or a code snippet within a list, use tabs to si
...
     <img src="{{site.url}}{{site.baseurl}}/images/Security/rule-dup2.png" alt="Opening the rule details pane" width="50%">
 
-1. Choose the **Duplicate** button in the upper-right corner of the pane. The Duplicate rule window opens in Visual Editor view, and all of the fields are automatically populated with the rule's details. Details are also populated in YAML Editor view.
+1. Select the **Duplicate** button in the upper-right corner of the pane. The Duplicate rule window opens in Visual Editor view, and all of the fields are automatically populated with the rule's details. Details are also populated in YAML Editor view, as shown in the following image.
```
Comment (natebower) [inline FORMATTING_GUIDE.md:306]: Should "Duplicate rule" be in bold as a UI element?

**Review by natebower — APPROVED** [review]
@kolchfa-aws Edits to the examples LGTM. Just two minor comments. Thanks!

File: FORMATTING_GUIDE.md (hunk 1 above)
Comment (kolchfa-aws) [reply FORMATTING_GUIDE.md]: Yes, updated.

File: FORMATTING_GUIDE.md (hunk 2 above)
Comment (kolchfa-aws) [reply FORMATTING_GUIDE.md]: Yes, updated.

**Review by carolxob — APPROVED** [review]

**Pull request merged**